Allay-Server/src/main/java/org/allaymc/server/world/AllayWorld.java

```diff
@@ -162,11 +167,13 @@ public void startTick() {
         }
     }
 
-    protected void tickTime(long tickNumber) {
+    protected void tickTime(long currentTick) {
         if (worldData.getGameRule(GameRule.DO_DAYLIGHT_CYCLE)) {
-            if (tickNumber >= nextTimeSendTick) {
+            if (currentTick >= nextTimeSendTick) {
                 worldData.setTime(worldData.getTime() + TIME_SENDING_INTERVAL);
-                nextTimeSendTick = tickNumber + TIME_SENDING_INTERVAL; // Send the time to client every 12 seconds
+                this.internalSkyLight = worldData.calculInternalSkyLight(currentTick);
```

Comment:
`internalSkyLight` update cannot be here for two reason:
- `calculInternalSkyLight` mehtod depend on time but also on rain and thunder strenght
- if gamerule `DO_DAYLIGHT_CYCLE` is desactivate, this line don't be called
Now, this is not a problem because rain and thunder are not implemented but when we will implemented this, internalSkyLight will bug

Reply:
Oh OK